### PR TITLE
Feature/refresh maintain session

### PR DESCRIPTION
**Pull request by axel-ledesma-david**
# Summary

- Create auth context.
- Create setSession function and refreshSession function where call to the endpoint auth/refresh.
- Create hook "useRefresh".
...

# Detail

Refresh user session calling to "auth/refresh" endpoint throught hook useRefresh.

### CODE REVIEW

File: src/components/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -19,11 +19,11 @@ export const LoginForm = () => {
 
     try {
       const response = await login(loginPayload);
-      console.log(response);
       toastAlertSuccess('Logeado satisfactoriamente');
       navigate('/', { replace: true });
+      console.log(response);
```
Comment (Fludeo) [inline src/components/LoginForm/LoginForm.tsx]: remove console log

File: src/pages/HomeTest.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* import { useContext, useEffect, useState } from "react";
+import { userInfo } from "../services/user.api";
+import { AuthContext } from "../context/AuthContext";
+import { IAuthContext } from "../types/types"; */
+
+const HomeTest = () => {
```
Comment (Fludeo) [inline src/pages/HomeTest.tsx]: We should not have this component in the repo. if you need it, keep it as a change in your local machine. 
Also, never submit commented code.

File: src/services/user.api.ts [hunk 3]
```diff
@@ -22,6 +26,27 @@ export const signup = async (payload: ISignUpPayload): Promise<unknown> => {
...
+export const refreshSessions = async (): Promise<{ accessToken: string }> => {
+  const res = await Axios.get('/auth/session', {
+    withCredentials: true,
+  });
+  return res.data;
+};
+
+export const setSession = async (token: string) => {
+  Axios.defaults.headers.common['Authorization'] = `Bearer ${token}`;
+  console.log(token, 'this is a token');
```
Comment (Fludeo) [inline src/services/user.api.ts]: remove this console log.

**Review by Fludeo — CHANGES_REQUESTED** [review]

File: src/components/LoginForm/LoginForm.tsx (hunk 1 above)
Comment (axel-ledesma-david) [reply src/components/LoginForm/LoginForm.tsx]: done

File: src/pages/HomeTest.tsx (hunk 2 above)
Comment (axel-ledesma-david) [reply src/pages/HomeTest.tsx]: done

File: src/services/user.api.ts (hunk 3 above)
Comment (axel-ledesma-david) [reply src/services/user.api.ts]: done

**Review by Fludeo — APPROVED** [review]
Great!!!

**Pull request merged**